bin/cli.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 /* eslint-disable no-console, no-process-env, max-statements */
+'use strict';
```

Comment:
wrt to your question about `use strict`, if you're writing CJS rather than ESM (which you are), you most likely want this to enforce strict mode for your files to prevent common mistakes.